components/Datepicker/src/datepicker.css

```diff
@@ -0,0 +1,220 @@
+  color: #7a7a7a;
+  appearance: none;
+  border-radius: 3px;
+  border: 1px solid #7a7a7a;
+  box-sizing: border-box;
+}
+
+.denhaag-datepicker__input::-webkit-calendar-picker-indicator {
+  display: none;
+}
+
+.denhaag-datepicker__input.denhaag-datepicker__input--has-value {
+  color: #4b4b4b;
+}
+
+.denhaag-datepicker--error .denhaag-datepicker__input {
+  border-color: #cf1c08;
+  padding-inline-end: 64px;
+}
+
+.denhaag-datepicker__input:hover {
+  border: 2px solid #1261a3;
+  padding-inline-start: 15px;
+  padding-inline-end: 47px;
+}
+
+.denhaag-datepicker--disabled .denhaag-datepicker__input {
+  color: #d2d2d2;
+  border-color: #d2d2d2;
+  pointer-events: none;
+}
+
+.denhaag-datepicker--error .denhaag-datepicker__input:hover {
+  padding-inline-end: 63px;
+}
+
+.denhaag-datepicker__error-icon {
+  display: none;
+}
+
+.denhaag-datepicker--error .denhaag-datepicker__error-icon {
+  position: absolute;
+  inset-block-start: 11px;
+  inset-inline-end: 40px;
+  display: block;
+  font-size: 18px;
+  color: #cf1c08;
+}
+
+.denhaag-datepicker--disabled .denhaag-datepicker__button {
+  color: #d2d2d2;
+  pointer-events: none;
+}
+
+.denhaag-datepicker .denhaag-datepicker__input.denhaag-datepicker__input--focus,
+.denhaag-datepicker .denhaag-datepicker__input:focus-visible,
+.denhaag-datepicker .denhaag-datepicker__button.denhaag-datepicker__button--focus + .denhaag-datepicker__input,
+.denhaag-datepicker .denhaag-datepicker__button:focus-visible + .denhaag-datepicker__input {
+  outline: var(--denhaag-focus-border);
+}
+
+.denhaag-datepicker__calendar {
+  position: absolute;
+  top: 40px;
+  left: 0;
+  padding-inline-start: 12px;
+  padding-inline-end: 12px;
+  padding-block-end: 16px;
+  border-radius: 3px;
+  border: 1px solid #d2d2d2;
+  box-sizing: border-box;
+}
+
+.denhaag-datepicker__calendar.denhaag-datepicker__calendar--hidden {
+  display: none;
+}
+
+.denhaag-datepicker__calendar-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+}
+
+.denhaag-datepicker__calendar-month {
+  font-family: "The Sans", sans-serif;
+  font-size: 18px;
+  line-height: 1.5;
+  font-weight: 500;
+}
+
+.denhaag-datepicker__calendar-navigation {
+  color: var(--denhaag-icon-button-color);
+
+  padding-inline-start: 0;
+  padding-inline-end: 0;
+  padding-block-start: 0;
+  padding-block-end: 0;
+
+  border: 0;
+  outline: 0;
+  border-radius: 50%;
+  cursor: pointer;
+  background-color: transparent;
+  appearance: none;
+}
+
+.denhaag-datepicker__calendar-navigation:hover {
+  color: var(--denhaag-icon-button-hover-color);
+}
+
+.denhaag-datepicker__calendar-navigation:focus {
+  color: var(--denhaag-icon-button-focus-color);
+  outline-color: var(--denhaag-icon-button-focus-outline-color);
+  outline-width: var(--denhaag-icon-button-focus-outline-width);
+  outline-style: var(--denhaag-icon-button-focus-outline-style);
+  outline-offset: var(--denhaag-icon-button-focus-outline-offset);
+}
```

Comment:
This focus state isn't documented in Figma. I added a [comment in figma](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn?node-id=1681:7756#135259358).

Also, wouldn't it make sense to use the IconButton component for this?

Reply:
Figma now documents the IconButton focus, which should also be rectangular. https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Design-System---Gemeente-Den-Haag?node-id=1681%3A7756

Reply:
Squared it is :(

Reply:
I tried using the `IconButton` here but I ran into some issues with the `onKeyDown` not working for our own  `IconButton` component. Don't want to spend too much time now debugging this, but nice to keep on our radar that when we remove MUI from the `IconButton` we should use it here.